File: Scripts/laptop/array-concatenation/php/array-concatenation.php
#!/usr/bin/php
<?php


$r = 1;

function executeTask($i) {

	$arr1 = array(1, 2, 3, 4, 5);
	$arr2 = array(6, 7, 8, 9, 10);
	$arr3 = array_merge($arr1, $arr2);
	return $i + 1;
}

for ( $i = 0; $i < 2000000000; $i++) {
	$r = executeTask($i + $r);
}

?>

File: Scripts/laptop/array-concatenation/r/array-concatenation.r

executeTask <- function(i) {
	a1 <- c(1+i, 2-i, 3+i, 4-i, 5+i)
	a2 <- c(6+i, 7-i, 8+i, 9-i, 10+i)
	a3 <- c(a1, a2)
	return(i)
}

r = 1
for(i in 0:2000000000) {
	r = executeTask(i)
}
